trunner/target/armv7a7.py

```diff
@@ -0,0 +1,122 @@
+from typing import Callable, Optional
+
+from trunner.dut import Dut, SerialDut
+from trunner.harness import (
+    HarnessBuilder,
+    PloInterface,
+    PloImageLoader,
+    PloImageProperty,
+    ShellHarness,
+    Rebooter,
+    RebooterHarness,
+)
+
+from trunner.harness import TerminalHarness
+from trunner.host import Host
+from trunner.tools import Phoenixd, Psu
+from trunner.types import TestResult, TestOptions
+from .base import TargetBase, find_port
+
+
+class ARMv7A7TargetRebooter(Rebooter):
+    def _set_flash_mode(self, flash):
+        self.host.set_flash_mode(not flash)
+
+
+class PsuPloLoader(TerminalHarness, PloInterface):
+    def __init__(self, dut: Dut, psu: Psu):
+        TerminalHarness.__init__(self)
+        PloInterface.__init__(self, dut)
+        self.psu = psu
+
+    def __call__(self):
+        """Loads plo image to RAM using psu tool."""
+        self.psu.run()
+        self.wait_prompt()
+
+        # Setting clean markers and erase space where jffs2 will be placed
+        self.send_cmd("jffs2 -d 2.0 -e -c 0x10:0x1f0:0x10000:0x10")
```

Comment:
Please add a jffs2 command in plo as it's done with `erase` and call it with appropriate arguments.